crates/common_utils/src/ext_traits.rs

```diff
@@ -219,13 +233,15 @@ impl ValueExt for serde_json::Value {
     where
         T: serde::de::DeserializeOwned,
     {
-        let debug = format!(
-            "Unable to parse {type_name} from serde_json::Value: {:?}",
-            &self
-        );
-        serde_json::from_value::<T>(self)
+        serde_json::from_value::<T>(self.clone())
```

Comment:
Is there an option for us to avoid the clone here and do the clone within the `attach_printable_lazy()` closure?

Reply:
We could do a `T::deserialize(&self)`, but reading through the serde documentation, passing an owned value allows serde to use a previously allocated buffer. If we pass a reference it cannot do so, ultimately defeating the purpose of removing the clone.

Apart from this I couldn't really find a way.